benchmarks/docker/django_simple/entrypoint

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -o errexit
+set -o pipefail
+set -o nounset
+
+# Install ddtrace
+if [[ -z "${DDTRACE_VERSION}" ]]; then
+    if [[ -d "/wheels" ]]; then
+        pip install --no-cache-dir --no-index --find-links=/wheels/ /wheels/*
+    else
+        pip install ddtrace
+    fi
+else
+    pip install ${DDTRACE_VERSION}
```

Comment:
From just the name, I would have assumed `DDTRACE_VERSION` was just a versions string like `0.49.2`

docs/examples or maybe a name change might help make it clear.

Reply:
I wonder if just installing from a wheel would be best for anything other than "most recently released"